cmd/check_container.go

```diff
@@ -81,12 +81,22 @@ var checkContainerCmd = &cobra.Command{
 }
 
 func init() {
-	checkContainerCmd.SetUsageTemplate(
-		`Usage:
+	checks := engine.ContainerPolicy()
+	checkString := "\n"
+	for _, value := range checks {
```

Comment:
you can replace this for loop with `strings.Join(engine.ContainerPolicy(), '-')`.

Reply:
That's cleaner. 
suggestion for adding a \n after each value?

Reply:
nvm, I added \n and that did the trick